mne/event.py

```diff
@@ -745,14 +745,20 @@ def make_fixed_length_events(raw, id, start=0, stop=None, duration=1.):
         of the recording.
     duration: float
         The duration to separate events by.
+    first_samp: bool
+        If True (default), times will be adjusted for first_samp. The default
+        behavior is desirable if times are arbitrary, as in resting data. Use 
```

Comment:
I don't quite get what you mean by this docstring. I think the default `first_samp=True` makes sense with (does the same thing as) `find_events`, where the event times have `raw.first_samp` built in. I would say something more like:
```
If True (default), times will have raw.first_samp added to them,
as in :func:`mne.find_events`. This is useful if events will be passed
to e.g. :class:`mne.Epochs`.
```